.git-blame-ignore-revs

```diff
@@ -1 +1,8 @@
-72819f91c19e076ca383e6e9fd7c7a510c62792e
```

Comment:
Out of interest? What was this? Maybe one of the ones we rebased and force pushed?

Reply:
A mistake I think.
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.